lib/tasks/import_images.rake

```diff
@@ -0,0 +1,10 @@
+namespace :import_images do
+  desc "Imports placeholder images using the Faker gem"
+  task generate_profile_pictures: :environment do
+    # Randomly grab half of all Authors, and set their profile images to Faker data;
+    Author.all.sample(Author.all.length/2).each do |author| 
```

Comment:
Wow, that is some great information. That is awesome, thank you very much. 

With that in mind, I'm definitely making some performance mistakes by calling `Author.all` _twice_.  If the dataset was larger, this might have been a very expensive rake task. I'll go ahead and push changes to the rake task. I swapped out `all.sample` for `first`, and `each` for `for_each`, leaving the default batch_size alone.  

Thanks! Keep an eye out for an updated branch.